src/blob/core.rs

```diff
@@ -92,12 +92,12 @@ where
         } else {
             self.fsyncdata()
                 .await
-                .with_context(|| "Blob file dump failed!")?;
+                .with_context(|| format!("Blob file dump failed! Blob file: {:?}", self.name.to_path()))?;
```

Comment:
Other errors start with lowercase, probably this one should be lowercase too. And also exclamation mark can be ommited in favor of comma IMO